lib/neo-test-functions.js

```diff
@@ -127,11 +127,15 @@
   const sendRequest = async (payload) => {
     console.info('INFO: Payload for request:', payload)
     replyId = null
+    let requestUrl = `https://${workspace}.neohelden.com/api/v1/particle`
+    if (typeof workspaceUrl !== 'undefined') {
```

Comment:
Wouldn't it be simpler to just check for definess?
```suggestion
    if (workspaceUrl !== undefined) {
```

Reply:
in most cases, workspaceUrl will not be defined; hence the typeof :)